pkg/ebpf/debugfs.go

```diff
@@ -37,6 +47,13 @@ type KprobeStats struct {
 // event name format is p|r_<funcname>_<uid>_<pid>
 var eventRegexp = regexp.MustCompile(`^((?:p|r)_.+?)_([^_]*)_([^_]*)$`)
 
+func RefreshProbeStats() {
+	for {
+		GetProbeStats()
+		time.Sleep(time.Duration(5) * time.Second)
```

Comment:
5 seconds seems aggressive. I think the default collection interval is 10 seconds.